src/NLog.Targets.Syslog/Settings/SdElementConfig.cs

```diff
@@ -13,13 +13,13 @@ namespace NLog.Targets.Syslog.Settings
     /// <summary>Syslog SD-ELEMENT field configuration</summary>
     public class SdElementConfig : NotifyPropertyChanged, IDisposable
     {
-        private SdIdConfig sdId;
+        private Layouts.SimpleLayout sdId;
         private ObservableCollection<SdParamConfig> sdParams;
         private readonly PropertyChangedEventHandler sdParamPropsChanged;
         private readonly NotifyCollectionChangedEventHandler sdParamsCollectionChanged;
 
         /// <summary>The SD-ID field of an SD-ELEMENT field in the STRUCTURED-DATA part</summary>
-        public SdIdConfig SdId
+        public Layouts.SimpleLayout SdId
```

Comment:
The implicit operator was present in my SdIdConfig, but I think that using SimpleLayout is better.
I simply would not prefix it with the `Layouts.` qualifier